test/utils/MaterializerSupport.scala

```diff
@@ -17,9 +17,7 @@
 package utils
 
 import akka.actor.ActorSystem
-import akka.stream.ActorMaterializer
 
```

Comment:
from what I can see we are still on akka 2.6.x, which is outside of scope of licensing changes they introduced recently (for more details => event-akka-licence-change channel); but we should be careful going forward as this anything above 2.7.x would required a license;

Reply:
Thus was needed to remove the compiler warnings